### DISCUSSION TITLE
TODOMVC in NiceGUI?

### DISCUSSION THREAD

**Discussion by metaperl** [zauberzeug/nicegui#923] [Ideas / Feature Requests]
[Solara has a TODOMVC implementation](https://solara.dev/examples/utilities/todo) that runs live in their docs. I would like to see how it compares with NiceGUI.

**Comment by metaperl** [comment]
Actually the TODOMVC example in solara [lives here](https://github.com/widgetti/solara/blob/master/solara/website/pages/examples/utilities/todo.py).

**Comment by rodja** [comment]
A ToDo example would be very nice. We currently use a `ToDo` data model in our [Trello drag and drop example](https://github.com/zauberzeug/nicegui/blob/main/examples/trello_cards/main.py) which should be changed to not have two "todo" apps...

**Comment by falkoschindler** [comment]
@metaperl I just created PR https://github.com/zauberzeug/nicegui/pull/954 showing a simple todo list based on the Solara example. (I left out the redundant edit dialogs.) Let me know what you think.

**Comment by falkoschindler** [comment]
With #954 released in 1.2.15 I'll close this discussion as completed.
